### PR TITLE
Make it possible to delete piped keys from the web

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

*Add delete menu item to the piped menu*
image

*Disable the menu item if old key is not exists.*
image

*Show alert if click add new key when an old key exists*
image

*Show information after added a new key*
Screen Shot 2021-05-12 at 16 13 14

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Make it possible to delete piped keys from the web
```

### CODE REVIEW

File: pkg/app/web/src/pages/settings/piped/index.tsx [hunk 1]
```diff
@@ -74,6 +74,9 @@ const selectFilteredPipeds = createSelector<
   }
 );
 
+const OLD_KEY_ALERT_MESSAGE =
+  "The old key is still there.\nTo add a new key, you need to delete the old one first.";
```
Comment (nghialv) [inline pkg/app/web/src/pages/settings/piped/index.tsx:78]: "The old key is still there.\nDo not forget to delete it once you update your Piped to use this new key."

File: pkg/app/web/src/pages/settings/piped/index.tsx [hunk 2]
```diff
@@ -190,7 +185,14 @@ export const SettingsPipedPage: FC = memo(function SettingsPipedPage() {
...
+          {registeredPiped?.isNewKey
+            ? "Added a new piped key"
+            : "Piped registered"}
+        </DialogTitle>
+        {registeredPiped?.isNewKey ? (
+          <Alert severity="info">{OLD_KEY_ALERT_MESSAGE}</Alert>
+        ) : null}
         <DialogContent>
           <TextWithCopyButton
             name="Piped Id"
```
Comment (nghialv) [inline pkg/app/web/src/pages/settings/piped/index.tsx:198]: Line 203: "Secret Key" -> "Piped Key", or "Key"

File: pkg/app/web/src/pages/settings/piped/piped-table-row.tsx [hunk 3]
```diff
@@ -169,15 +206,36 @@ export const PipedTableRow: FC<Props> = memo(function PipedTableRow({
...
             </MenuItem>,
             <MenuItem key="piped-menu-disable" onClick={handleDisable}>
               {UI_TEXT_DISABLE}
             </MenuItem>,
           ]
         )}
       </Menu>
+
+      <Dialog open={openOldKeyAlert} onClose={handleAlertClose}>
+        <DialogTitle>An old key exists</DialogTitle>
```
Comment (nghialv) [inline pkg/app/web/src/pages/settings/piped/piped-table-row.tsx:227]: "There are already 2 keys for this piped"

**Comment by cakecatz** [comment]
@nghialv Thank you. I fixed the text as you proposed.

**Comment by khanhtc1202** [comment]
Way to go 🚀 Thanks
/approve

**Comment by nakabonne** [comment]
@cakecatz You've done a fantastic job. A big thank you goes out to you!

**Pull request merged**